python/paddle/v2/fluid/profiler.py

```diff
@@ -8,9 +9,20 @@
     "gpuendtimestamp",
     "gridsize3d",
     "threadblocksize",
+    "dynsmemperblock",
+    "stasmemperblock",
+    "regperthread",
+    "memtransfersize",
+    "memtransferdir",
+    "memtransferhostmemtype",
     "streamid",
+    "cacheconfigrequested",
+    "cacheconfigrequested",
+    "cacheconfigrequested",
     "enableonstart 0",
     "conckerneltrace",
+    "active_warps",
+    "active_warps",
```

Comment:
Why "active_warps" appeared two times, and "cacheconfigrequested"  appeared three times.

Reply:
Fix, Thanks!